runtime/src/main/cpp/Memory.cpp

```diff
@@ -1554,6 +1489,16 @@ void garbageCollect(MemoryState* state, bool force) {
   GC_LOG("<<< GC: toFree %d toRelease %d\n", state->toFree->size(), state->toRelease->size())
 }
 
+void rememberNewContainer(ContainerHeader* container) {
```

Comment:
This operation should also be applied to every reference read from the "heap" that could be released by other thread, e.g.

* References from `StableRef`
* References from `KRefSharedHolder`
* Values of `MutableData.buffer`

Reply:
+ `Konan_WeakReferenceCounter_get`

Not sure if I found all such references.